src/App.jsx

```diff
@@ -25,21 +26,16 @@ export const alex = {
 
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
+    <Person
+      person={misha}
+    />
```

Comment:
there is no need to put an attribute on a new line.
write all component on the same line